dashboard/modules/reporter/reporter_head.py

```diff
@@ -119,8 +119,8 @@ async def get_cluster_status(self, req):
         """Returns status information about the cluster.
 
         Currently contains two fields:
-            autoscaling_status (str): a status message from the autoscaler.
-            autoscaling_error (str): an error message from the autoscaler if
+            autoscaling_status: a status message from the autoscaler.
```

Comment:
I think we don't want to remove the type annotation here since it is a field within `req`? (It is also not Args field)